templates/types/streaming/express/src/controllers/llamaindex-stream.ts

```diff
@@ -30,15 +30,12 @@ function createParser(
           },
         };
         data.append(message);
-      } else {
-        data.append({}); // send an empty image response for the user's message
```

Comment:
@thucpn I remember I added this to ensure that the client always receives a data object for each message. That way it's easy to map the message with the data object. 
I don't think images will work if we remove this